### PR TITLE
ITD player cards (spoilers)

### PR DESCRIPTION
**Pull request by zzorba**

### CODE REVIEW

File: pack/tic/itd.json [hunk 1]
```diff
@@ -1,4 +1,56 @@
...
+		"flavor": "\"If you blow that thing one more time, I'm gonna make you eat it.\"",
+		"illustrator": "Robert Laskey",
+		"name": "Riot Whistle",
+		"pack_code": "itd",
+		"position": 108,
+		"quantity": 2,
+		"skill_willpower": 1,
+		"slot": "Accessory",
+		"text": "You may take an additional action during your turn, which can only be used to <b>engage</b>.",
+	    "traits": "Item. Tool.",
```
Comment (drawntotheflame) [inline pack/tic/itd.json:16]: NB this line and xp line aren't tabbed in to the same length as the rest.

File: pack/tic/itd.json [hunk 2]
```diff
@@ -1,4 +1,56 @@
...
+		"type_code": "asset",
+	    "xp": 0
```
Comment (drawntotheflame) [inline pack/tic/itd.json:18]: NB indent.

File: pack/tic/itd.json [hunk 3]
```diff
@@ -1,4 +1,56 @@
...
+	},
+	{
+		"code": "07109",
+		"cost": 1,
+		"deck_limit": 2,
+		"faction_code": "guardian",
+		"flavor": "Lita narrowed her eyes and tightened her grip. They wouldn't go away. Not this time. Not ever again.",
```
Comment (drawntotheflame) [inline pack/tic/itd.json:25]: Typo: 'go' instead of 'get' ('They wouldn't get away')

File: pack/tic/itd.json [hunk 4]
```diff
@@ -1,4 +1,56 @@
...
+		"code": "07110",
+		"cost": null,
+		"deck_limit": 1,
+		"faction_code": "guardian",
+		"illustrator": "Tiziano Baracchi",
+		"name": "Sacred Covenant",
+		"pack_code": "itd",
+		"permanent": true,
+		"position": 110,
+		"quantity": 1,
```
Comment (drawntotheflame) [inline pack/tic/itd.json:48]: Are you confident on the quantities of the covenants? I haven't got my pack yet to check...

File: pack/tic/itd.json [hunk 5]
```diff
@@ -1,4 +1,56 @@
...
+		"type_code": "asset",
+		"traits": "Covenant. Blessed.",
```
Comment (drawntotheflame) [inline pack/tic/itd.json:50]: Type code, traits and text are in inverse alphabetical order.

File: pack/tic/itd.json [hunk 6]
```diff
@@ -1,4 +1,56 @@
...
+		"text": "Permanent. Limit 1 [[Covenant]] per deck.\n[reaction] After an investigator at any location performs the \"reveal chaos token\" step of a skill test, exhaust Scred Covenant: Return any number of [bless] tokens revealed during this test to the chaos bag, ignoring their modifiers for this test.",
```
Comment (drawntotheflame) [inline pack/tic/itd.json:51]: Typo: 'Sacred' instead of 'Scred'.

File: pack/tic/itd.json [hunk 7]
```diff
@@ -19,6 +71,24 @@
...
+		"flavor": "When you start following a thread, it's hard to say what you'll find at the other end.",
+		"illustrator": "Winona Nelson",
+		"name": "Stirring Up Trouble",
+		"pack_code": "itd",
+		"position": 112,
+		"quantity": 2,
+		"skill_combat": 1,
+		"skill_intellect": 1,
+		"text": "As an additional cost to play Stirring Up Trouble, add [curse] tokens to the chaos bag equal to your location's shroud value.\nDiscover 2 clues at your location.",
+		"type_code": "event",
```
Comment (drawntotheflame) [inline pack/tic/itd.json:88]: NB type code comes after traits.

File: pack/tic/itd.json [hunk 8]
```diff
@@ -84,6 +189,39 @@
...
+		"deck_limit": 2,
+		"faction_code": "mystic",
+		"illustrator": "Patrick McEvoy",
+		"name": "Shroud of Shadows",
+		"pack_code": "itd",
+		"position": 119,
+		"quantity": 2,
+		"skill_agility": 1,
+		"slot": "Arcane",
+		"text": "Uses (3 charges).\n[Action] Spend 1 charge: <b>Evade.<b/> This evasion uses [willpower] instead of [agility]. If you succeed and the evaded enemy is non-[[Elite]], you may move that enemy to a connecting location. If a [curse] token is revealed during this evasion attempt, you may move to a connecting location or place 1 charge on Shroud of Shadows.",
```
Comment (drawntotheflame) [inline pack/tic/itd.json:204]: NB 'Action' is capitalised when it doesn't need to be.

File: pack/tic/itd.json [hunk 9]
```diff
@@ -84,6 +189,39 @@
...
+		"code": "07120",
+		"cost": null,
+		"deck_limit": 1,
+		"faction_code": "mystic",
+		"illustrator": "Tiziano Baracchi",
+		"name": "Paradoxical Covenant",
+		"pack_code": "itd",
+		"permanent": true,
+		"position": 120,
+		"quantity": 1,
```
Comment (drawntotheflame) [inline pack/tic/itd.json:219]: NB quantity.

File: pack/tic/itd.json [hunk 10]
```diff
@@ -84,6 +189,39 @@
...
+		"text": "Permanent. Limit 1 [[Covenant]] per deck.\n[reaction] After an investigator at your location performs the \"reveal the chaos token\" step of a skill test, if both a [bless] and a [curse] token were revealed during that test, exhaust Paradoxical Covenant: This test automatically succeeds. <i>(Remove each [bless] and [curse] token revealed from the chaos bag after this test ends.)</i>",
```
Comment (drawntotheflame) [inline pack/tic/itd.json:220]: missing word: it should read 'if both a [bless] token and a [curse] token', repeating the word token.

**Review by drawntotheflame — COMMENTED** [review]
Great job @zzorba

File: pack/tic/itd.json (hunk 4 above)
Comment (zzorba) [reply pack/tic/itd.json:48]: Yeah, all reports are there are only 1 of each (hence why we get 15 player cards instead of the usual 10)

**Comment by zzorba** [comment]
All corrections completed, thank you @drawntotheflame.

@Kamalisk this can be merged now.

**Pull request merged**